<div align="center">

# 1º PROJETO

| Disciplina | Semestre | Docente | Horário |
| :---: | :---: | :---: | :---: |
| PAM0466 - SISTEMAS INTELIGENTES | 2026.1 | PEDRO THIAGO VALÉRIO DE SOUZA | 3M23 4M45 |

| Discente | Matrícula |
| :---: | :---: |
| ELTON CAIO VIEIRA DE LIMA | 2020010673 |
| LUCAS VIERES ARAÚJO FARIAS | 2025022531 |
</div>

<div align="center">

## Atividades
</div>

1. Calcule a correlacão entre as variáveis independentes (AT, V, AP, RH) e a variável dependente (PE). Quais variáveis parecem ter maior influência na producão de energia?

In [10]:
import pandas

dataset = pandas.read_csv("./datasets/Power Plant Data.csv")
dataset

x = dataset[["AT", "V", "AP", "RH"]]
y = dataset["PE"]

corr = x.corrwith(y)
print(corr)

AT   -0.948128
V    -0.869780
AP    0.518429
RH    0.389794
dtype: float64


Pela interpretação dos valores, as variáveis independentes AT e V apresentam uma correlação muito forte (de magnitudes 0.948128 e 0.869780, respectivamente) com a variável dependente PE, mas inversamente proporcionais (sinal negativo).

Por outro lado, RH e AP apresentam correlações moderada e forte (magnitudes 0.389794 e 0.518429), ambas diretamente proporcionais (sinal positivo).
<hr>

2. Calcule a correlacão entre as variáveis independentes (AT, V, AP, RH) entre si. Existe alguma relacão forte entre elas? Se sim, isso pode indicar multicolinearidade.

In [11]:
print(x.corr())

          AT         V        AP        RH
AT  1.000000  0.844107 -0.507549 -0.542535
V   0.844107  1.000000 -0.413502 -0.312187
AP -0.507549 -0.413502  1.000000  0.099574
RH -0.542535 -0.312187  0.099574  1.000000


As variáveis independentes AT e V apresentam uma correlação muito forte (de magnitude 0.844107) e diretamente proporcional (sinal positivo).
<hr>